Tests/RegressionTests.cs

```diff
@@ -412,25 +412,25 @@ private static TestCaseData[] GetRegressionTestParameters()
 
             var coarseFineFundamentalRegressionAlgorithmStatistics = new Dictionary<string, string>
             {
-                {"Total Trades", "6"},
+                {"Total Trades", "1"},
```

Comment:
Changes derive from the date range change as well as adding a condition that the 1 year EPS growth be greater than `0.25m`